DRAFT_slitasje.Rmd

```diff
@@ -0,0 +1,1496 @@
+    values to justify the extrapolation
+
+10. Sample 10 00 random points from an ecosystem delineation and extract
+    indicator values and sd at those points.
+
+11. Export indicator maps and regional extrapolated maps
+
+## About the underlying data
+
+The indicator uses a data set from a standardised field survey of nature
+types. You can read more about the preliminary analyses
+[here](#naturtype). See also the [official
+site](https://www.miljodirektoratet.no/ansvarsomrader/overvaking-arealplanlegging/naturkartlegging/naturtyper/)
+of the Environmental Agency. I also import a data set called
+[ANO](#ANO-import), which you can read about
+[here](https://www.miljodirektoratet.no/ansvarsomrader/overvaking-arealplanlegging/miljoovervaking/overvakingsprogrammer/natur-pa-land/arealrepresentativ-naturovervakning-ano/)
+
+### Representativity in time and space
+
+The nature type mapping is not random and cannot be said to be area
+representative. The ANO dataset however, is area representative. The
+data is from 2018 to present. The data from one field season usually
+becomes available early the following year.
+
+### Original units
+
+The variables are recorded on a coarse four-sped ordinal scale (Fig.
+\@ref(fig:four-step)).
+
+### Temporal coverage
+
+The data goes back to 2018, but is because of data shortage, it is
+bulked into one time step. I then use the mean date for the raw data,
+and define the variable as belonging to the year 2020 (read more
+[here](#scaled-slitasje-variable)).
+
+### Aditional comments about the dataset
+
+For a run through of the data set, see [here](#naturtype).
+
+## Ecosystem characteristic
+
+### Norwegain standard
+
+The indicator is tagged to the *Økologisk egenskap* called
+**Primærproduksjon** (Primary productivity). This is tentative, and
+perhaps *abiotiske forhold* is more suited. But the thought behind the
+chiose is that *sliatsje* affect the potential for primary productivity.
+
+### UN standard
+
+The indicator is tagged as a **Physical state characteristics**
+indicator. This is quite clear. It's a tpe of abiotic characteritic that
+has to do with the physical structure rather than the chemical
+composition.
+
+## Collinearities with other indicators
+
+Slitasje is not thought to be very collinear with any other indicator at
+the present.
+
+## Reference condition and values
+
+### Reference condition
+
+The reference condition is one with minimal negative human impact. This
+is also true for semi-natural ecosystems. For example, in the reference
+condition, *slitasje* in semi-natural ecosystems is defined to be zero,
+even though at no point in time would this condition be realized.
+
+### Reference values, thresholds for defining *good ecological condition*, minimum and/or maximum values
+
+The upper reference value is 0 (no *slitasje*), and the lower reference
+value is 100%. Note that 100% slitasje doesn't mean that all the area
```

Comment:
"does not" (doesn't is "spoken English" written!)

Reply:
very good